src/ontology/mondo-edit.obo

```diff
@@ -435350,21 +435336,35 @@ is_a: MONDO:0019250 {source="Orphanet:79169"} ! inborn disorder of biogenic amin
 [Term]
 id: MONDO:0019220
 name: inborn disorder of cobalamin metabolism and transport
+def: "An inherited metabolic disease affecting cobalamin (vitamin B12) intestinal absorption, transport in the blood, uptake by peripheral cells or cellular metabolism." [PMID:35337626]
 subset: clingen {source="MONDO:CLINGEN"}
 subset: disease_grouping
 subset: gard_rare {source="GARD:18951", source="MONDO:GARD"}
+subset: gard_rare {source="MONDO:GARD"}
```

Comment:
this line is duplicated

Reply:
removed duplicated line.

Reply:
I forgot to tell you (sorry! I got distracted). 
The easy way to deal with things like this is by running a normalization step